src/auth/screens/auth-profile.screen.js

```diff
@@ -113,71 +114,87 @@ class AuthProfile extends Component {
     }
   };
 
+  refreshProfile = () => {
+    this.props.getUser();
```

Comment:
Can we extract this to a separate method (for example: `getUserDetails`) and then call `getUserDetails` in `componentDidMount` as well as in the `ScrollView` `refreshControl` prop?